test/functional/helpers/createExtensionViewController.js

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+module.exports = viewPath => {
```

Comment:
Sweet!

@Aaronius is there a doc somewhere around the `loadExtensionView` API?

Reply:
Only the doc I linked to in the PR description (https://www.npmjs.com/package/@adobe/reactor-sandbox#using-the-sandbox-for-automated-testing).

Reply:
I'll add a comment in the code with the URL.